_sass/layout/_l-content.scss

```diff
@@ -52,10 +52,83 @@
     margin-top: rem(70px);
   }
 
```

Comment:
This should go in its module, not in l-content
l-content is a very general layout